tests/pathfinding/test_blockchain_integration.py

```diff
@@ -38,12 +38,13 @@ def test_pfs_with_mocked_client(
         registry_address=token_network_registry_contract.address,
         required_confirmations=1,
         poll_interval=0,
+        private_key='3a1076bf45ab87712ad64ccb3b10217737f7faacbf2872e88fdd9a537d8fe266',
     )
 
     # greenlet needs to be started and context switched to
     pfs.start()
     wait_for_blocks(1)
-    gevent.sleep(0)
+    gevent.sleep(1)
```

Comment:
Is this still necessary after the other changes?

Reply:
tests fail when it is removed. So it seems necessary. I suggest to keep it